packages/evm/CHANGELOG.md

```diff
@@ -6,20 +6,89 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-- VM access to `StateManager` is now all done using the EEI.
-- Internally, in the EVM, the `Env` environment variable is used to track any variables which do not change during a call frame, for instance the `code`, the `caller`, etc. The `RunState` is used to track anything which can change during the execution, such as the remaining gas, the selfdestruct lists, the stack, the program counter, etc.
-- The EVM is now fully typed. Before, the `AsyncEventEmitter` did not have an interface, therefore TypeScript internally casts it as `any`. It also provides types for the available events.
-- EVM provides the interface for `EEI` and `EVM`, which can be used to create a new EVM/EEI and use this in the VM.
-- TransientStorage is now part of EVM and not of EEI.
-- Renamed `gasUsed` to `executionGasUsed` as part of `ExecResult`.
+We are very much confident that users of the libraries will greatly benefit from the changes being introduced. However - along the upgrade process - these releases require some extra attention and care since the changeset is both so big and deep reaching. We highly recommend to closely read the release notes, we have done our best to create a full picture on the changes with some special emphasis on delicate code and API parts and give some explicit guidance on how to upgrade and where problems might arise!
+
+So, enjoy the releases (this is a first round of Beta releases, with final releases following a couple of weeks after if things go well)! 🎉
+
+The EthereumJS Team
+
+### New Package
+
+With this release there is now a dedicated `@ethereumjs/evm` package extracted from the `VM` (or: `@ethereumjs/vm` package), see PRs [#1892](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1892), [#1955](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1955) and [#1977](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1977) for the main implementation work and PR [#1974](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1974) for the package extraction work. The new package can be installed with:
+
+```shell
+npm i @ethereumjs/evm
+```
+
+(please note that atm this package is not yet completely standalone but still needs the outer VM package to have some useful context to run. Functionality to have the EVM completely standalone will be added later on in a non-breaking way,)
+
+The new EVM package extracts the bytecode execution logic and leaves the handling of the outer environment like setting up some pre-state, processing txs and blocks, generating receipts and paying miners (on a PoW chain) to the outer package.
+
+This makes up for a cleaner separation of concerns, generally the new packages comes a lot closer to being a pure bytecode execution Ethereum Virtual Machine (EVM) implementation than before and will allow for new ways of both customizing and adopting the inner EVM as well as providing an alternative environmental context and customize on the outer processing by adopting the outer VM package.
```

Comment:
```suggestion
This makes for a cleaner separation of concerns and generally brings the the new packages a lot closer to being a pure bytecode execution Ethereum Virtual Machine (EVM) implementation than before.  This will allow for new ways of both customizing and adopting the inner EVM as well as providing an alternative environmental context and customize on the outer processing used in the outer VM package.
```